src/example13-opacity-raf.js

```diff
@@ -1,21 +1,43 @@
-var frames_remaining = 120;
-var go_down = true;
 var div = document.querySelectorAll("div")[0];
-function animate() {
-    var percent_remaining = frames_remaining / 120;
-    if (!go_down) percent_remaining = 1 - percent_remaining;
-    div.style = "opacity:" +
-        (percent_remaining * 0.999 +
-            (1 - percent_remaining) * 0.1);
-    if (frames_remaining-- == 0) {
-        go_down = !go_down
-        frames_remaining = 120;
-    }
-    return true;
+var total_frames = 120;
+var current_frame = 0;
+var change_per_frame = 0.899 / total_frames;
```

Comment:
```suggestion
var change_per_frame = 0.999 / total_frames;
```

Reply:
The animation is now from 0.1 to 0.999. I've clarified this in the code (replacing 0.899 with `(0.999 - 0.1)`).